### PR TITLE
Internationalization of plugin and Croogo::hookRoutes

### PR DESCRIPTION
**Pull request by John-Henrique**
See this changes

### CODE REVIEW

File: controllers/eshop_suppliers_controller.php [hunk 1]
```diff
@@ -14,6 +14,31 @@ class EshopSuppliersController extends EshopAppController {
          * @var string
          */
         public $name = 'EshopSuppliers';
+        
+
```
Comment (elcuro) [inline controllers/eshop_suppliers_controller.php:18]: List of eshop suppliers can not be public, is only for eshop admin

File: controllers/eshop_basket_controller.php [hunk 2]
```diff
@@ -68,16 +68,26 @@ class EshopBasketController extends EshopAppController {
...
-                        $this->EshopItem->unbindModel(
-                                array('belongsTo' => array('EshopSupplier'))
-                        );
-			$data = $this->EshopItem->findById($item_id);
-                        $data['count'] = $item_count;
-                        $items[] = $data;
-		}
+                
+                
+                // evita o erro com carrinho vazio
```
Comment (elcuro) [inline controllers/eshop_basket_controller.php]: pls change all notices to english

**Comment by elcuro** [comment]
Hi,
nice job,
i wrote some notices on github before merge

2011/7/26 hostdesigner
reply@reply.github.com:

File: config/eshop_routes.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?php
```
Comment (elcuro) [inline config/eshop_routes.php:1]: will be better to have all urls in english, same as every where in plugin

Comment (John-Henrique) [inline config/eshop_routes.php:1]: Yes, I mistake, next fix

File: controllers/eshop_basket_controller.php (hunk 2 above)
Comment (John-Henrique) [inline controllers/eshop_basket_controller.php]: Notices or comment lines? The notices of setFlash area translated if exists a localized info...

About comments line... fix

File: controllers/eshop_suppliers_controller.php (hunk 1 above)
Comment (John-Henrique) [inline controllers/eshop_suppliers_controller.php:18]: This is a simple modification to show products of brandy suppliers, yet no completed

File: config/eshop_routes.php (hunk 3 above)
Comment (John-Henrique) [inline config/eshop_routes.php:1]: Seria interessante localizar o URI? Would prevent the user need to edit the eshop_routes.php
Example:
CroogoRouter::connect( __(  'eshop', '/shop/cart', true ), array() );

Comment (John-Henrique) [inline config/eshop_routes.php:1]: Do this CroogoRouter::connect( __( 'eshop', '/shop/cart', true ), array() ); and works fine CakePHP and Croogo are very flexible

File: controllers/eshop_basket_controller.php (hunk 2 above)
Comment (John-Henrique) [inline controllers/eshop_basket_controller.php]: I fix this

File: controllers/eshop_suppliers_controller.php (hunk 1 above)
Comment (elcuro) [inline controllers/eshop_suppliers_controller.php:18]: For grouping by  manfuacturers or brands is croogo inbuild taxonomy  better way.
Supplier (in this case) is usable for bulk collection of products from suppliers wholesale (api, xls sheets, etc.)

File: controllers/eshop_basket_controller.php (hunk 2 above)
Comment (elcuro) [inline controllers/eshop_basket_controller.php]: comments

**Pull request closed without merging**